dsgrid/config/dataset_config.py

```diff
@@ -103,6 +103,12 @@ class DataSchemaType(DSGEnum):
         Typically appropriate for small, low-temporal resolution datasets.
         """,
     )
+    GROWTH_RATE = EnumValue(
+        value="growth_rate",
+        description="""
+        One_table data schema with load_data table for unitless and temporally arbitrary data, such as growth rates.
+        """,
+    )
```

Comment:
I was thinking that growth rates would use the same schemas as before. They'd just contain growth rate (fractional) data instead of, e.g., energy or stock data.